base/utils/R/combine_model_netcdf.R

```diff
@@ -0,0 +1,218 @@
+#' Merge model outputted netCDF files by time steps specified by pecan settings file.
+#' @details
+#' The function is only tested for SIPNET model runs.
+#' Please make sure you have the same netCDF formats if you want to proceed with different models.
+#' We could also have more functions that deal with different dimensions (e.g., by site instead of by year).
+#' 
+#' @param settings.dir character: physical path to the pecan standard multi-settings file.
+#' @param nc.outdir  character: physical path to the folder that contains the merged netCDF files.
+#' @param ancillary.inputs list: necessary arguments if settings.dir is NULL. See 
+#' `model.outdir` path to the folder that contains model outputs; 
+#' `ens.num` number of ensembles for the model run;
+#' `site.ids` vector of site ids across locations;
+#' `cores` number of CPUs for the parallel computation;
+#' `start.date` start date of the model run;
+#' `end.date` end date of the model run.
+#' `time.step` time step of the model run (e.g., 1 year).
+#'
+#' @return character: file paths to the merged netCDF files.
+#' @export
+#' 
+#' @author Dongchen Zhang
+#' @importFrom purrr %>%
```

Comment:
Since other functions in this package are already importing %>% from magrittr, let's use that one 

(Sorry @DongchenZ, I know I've made you switch this the other direction in other PRs 😅. My reasoning is just that we want to minimize new imports at any given moment )
```suggestion
#' @importFrom magrittr %>%
```

Reply:
Fixed.